pkg/ebpf/c/common/filtering.h

```diff
@@ -189,6 +203,10 @@ statfunc u64 compute_scopes(program_data_t *p)
         return 0;
     }
 
+    //
+    // unversioned filters
```

Comment:
This is confusing for me. Is unversioned filters that aren't part of the `pols_version`? You did explain to me off line that it isn't the case, so perhaps improve the comments here?

Reply:
Let's change those comments for something like: `// equality filters (using versioned maps)` and `boolean filters (not using versioned maps)`

Reply:
@josedonizetti this is solved by: https://github.com/aquasecurity/tracee/pull/3735/commits/ec9c7e5718f46f135c05209341d5be380570be24